deploy/role.yaml

```diff
@@ -15,6 +15,27 @@ rules:
   - secrets
   verbs:
   - "*"
+- apiGroups:
+  - ""
+  resources:
+  - podtemplates
+  - namespaces
+  - componentstatuses
+  - nodes
+  - resourcequotas
+  - serviceaccounts
+  - processedtemplates
+  - replicationcontrollers
+  - limitranges
+  verbs:
+  - get
+  - list
+- apiGroups: 
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
```

Comment:
@xcoulon, I don't think we need to put it in role, as it's clusterscoped resource, and it's already added in `cluster_role.yaml

Reply:
good catch, indeed. I missed the `cluster_role.yaml` file. So I guess the problem we've seen is due to something else, then

Reply:
so let me revert the last 2 commits